example/gardener-local/gardenlet/values-seed2.yaml

```diff
@@ -0,0 +1,30 @@
+global:
```

Comment:
Maybe rename this to `values-kind2.yaml` and rename the existing `values.yaml` to `values-kind.yaml`?

Reply:
Was wondering the same, but `values-kind2.yaml` is something that we add on top (and together with) `values.yaml` So both are used for the second seed. I think it would be better to leave them as it is.

Reply:
OK, fair enough. Then still `values-seed2.yaml` -> `values-kind2.yaml` would be great